csvReader/csvBenchmark_test.go

```
@@ -0,0 +1,60 @@
+package csvReader
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"os"
+	"testing"
+)
+
+//func TestBench(t *testing.T) {
+//	fn := "big.csv.gz"
+//	buildCsvBigFile(fn)
+//	runner(t, fn, "plain", 1000000)
+//}
+
+func TestMain(m *testing.M) {
+	fn := "big.csv.gz"
+	buildCsvBigFile(fn)
+
+}
+
+func BenchmarkReading(t *testing.B) {
+	fn := "big.csv.gz"
+	//	buildCsvBigFile(fn)
+	runnerB(t, fn, "plain", 1000000)
+}
+
+// create a 6.1M File
+func buildCsvBigFile(tmpFile string) {
+	//var w io.Writer
+	iow, _ := os.Create(tmpFile)
```

As it is a test you should assert that error is `nil` instead of silencing.

```go
iow, err := os.Create(tmpFile)
if err != nil {
    t.Error(e.Error())
    return false
}
```
**As a rule of thumb:** You should **NEVER** let an error get silenced.